server/apps/research/views.py

```
@@ -42,7 +43,9 @@
             return Response(serializer.data, status=status.HTTP_201_CREATED)
         except Exception as e:
             logger.error(f"Unexpected error during article creation: {e}")
-            return Response({'error': 'An unexpected error occurred'}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+            if isinstance(e, serializers.ValidationError):
```

Ruff (0.8.0)

Undefined name `serializers`

(F821)